### PR TITLE
fixed examples

### PR DESCRIPTION
**Pull request by claasahl**
Attempted to parse/process YML files (in [./documentation/definitions/](https://github.com/scrive/scrive-apidocs/tree/master/documentation/definitions)) with [yamljs](https://www.npmjs.com/package/yamljs/v/0.3.0).

It turned out that some examples were rejected due to their use of white-space.

---
This change is [Reviewable](https://reviewable.io/reviews/scrive/scrive-apidocs/53)

### CODE REVIEW

**Comment by wwwater** [comment]
you can run it locally and see how it looks like with `bundle exec middleman server`

File: documentation/definitions/DataRetentionPolicies.yaml [hunk 1]
```diff
@@ -4,17 +4,19 @@ title: DataRetentionPolicies
...
+      "immediate_trash":false
+    },
+    "data_retention_policy": {
+      "idle_doc_timeout_canceled":45,
+      "immediate_trash":true,
+      "idle_doc_timeout_error":67,
+      "idle_doc_timeout_preparation":32
+    }
+  }
+  properties:
```
[the remark this answers is not in the document]
Comment (claasahl) [reply documentation/definitions/DataRetentionPolicies.yaml]: you are absolutely right

**Comment by claasahl** [comment]
thanks for the feedback, @wwwater

**Comment by claasahl** [comment]
I switched YML parsers. Now, I am using [js-yaml](https://www.npmjs.com/package/js-yaml) instead of  [yamljs](https://www.npmjs.com/package/yamljs/v/0.3.0).

**Pull request closed without merging**